bin/newman.js

```diff
@@ -62,11 +62,64 @@ program
     .option('--export-cookie-jar <path>', 'Exports the cookie jar to a file after completing the run')
     .option('--verbose', 'Show detailed information of collection run and each request sent')
     .action((collection, command) => {
+        // Process reporter arguments explicitly
+        const rawArgs = command.parent.args,
+            isReporterPresent = rawArgs.includes('-r') || rawArgs.includes('--reporter');
+
+        if (isReporterPresent) {
+            const reporters = ['cli'];
+            let reporterIndex = rawArgs.indexOf('-r'),
```

Comment:
Instead of reassigning `reporterIndex` again (in case of `--reporter`), I think we can do this:
```js
const reporterIndex = rawArgs.indexOf('-r') !== -1 ? rawArgs.indexOf('-r') : rawArgs.indexOf('--reporter')
```

Or, compromising readability a bit, the following:

```js
const reporterIndex = Math.max(rawArgs.indexOf('-r'), rawArgs.indexOf('--reporter'))
```

P.S.
A question: What happens in the case of both `-r` and `--reporter` being passed?

Reply:
Thanks for the suggestion, but statement 1 gives us a es-lint error and no compromises can be made on readability.
Second, this PR is redundant as we have found a better way around this so I'd be closing this.
😄

Reply:
Yeah. Just saw that we're going with variadic args, which imo, is a better choice.  🎉